source/adios2/core/ADIOS.cpp

```diff
@@ -27,18 +28,83 @@
 #include "adios2/operator/callback/Signature1.h"
 #include "adios2/operator/callback/Signature2.h"
 
+#ifdef ADIOS2_HAVE_AWSSDK
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/LogLevel.h>
+Aws::SDKOptions awdSDKOptions;
+#endif
+
 namespace adios2
 {
 namespace core
 {
 
+class ADIOS::GlobalServices
+{
+public:
+    GlobalServices() { std::cout << "ADIOS Global() " << std::endl; }
```

Comment:
Lurking std::cout's?

Reply:
Yeah, I have plenty of those. This one at least ruins the Skeleton tests so no merging is possible.